[v6r22] Update transformationplugin.rst (#4187)
DIRACGrid/DIRAC: docs/source/DeveloperGuide/Systems/Transformation/transformationplugin.rst

The plugins that can be used inside the Transformation System, need to be added in the list of allowed plugins in the

> ``Operations/Transformations/AllowedPlugins`` option in the Configuration System::

Reviewer: ```suggestion
 ``Operations/Transformations/AllowedPlugins`` option in the Configuration System.
```

> If there is already the ``AllowedPlugins`` defined, simply add the new plugin::

Reviewer: ```suggestion
If the option ``AllowedPlugins`` is already defined, simply add the new plugin::
```

  Transformations
  {
    AllowedPlugins = ...
    AllowedPlugins += ...
    AllowedPlugins += MyNewPlugin
  }
  

> Or, if it is not defined, you need to add it with the default plugins in the code, otherwise the other plugins would stop working::

Reviewer: ```suggestion
Or, if it is not defined, you need to add the ``AllowedPlugins`` option including the list of default plugins, otherwise the other plugins would stop working::
```
